kubernetes/charts/weblogic-operator/templates/_operator-dep.tpl

```diff
@@ -53,6 +53,10 @@ spec:
           value: "false"
         - name: "JAVA_LOGGING_LEVEL"
           value: {{ .javaLoggingLevel | quote }}
+        - name: "JAVA_LOGGING_MAXSIZE"
+          value: {{ .javaLoggingFileSizeLimit | default 1000000 | quote }}
```

Comment:
For Kubernetes, you get by default the last 10 MB of the pod output. Here, we have seven files each of 1 MB. I could match the Kubernetes behavior by changing the number of files from 7 to 10.

Reply:
I don't really feel strongly about these defaults, so I'm happy to adjust to the group consensus.

Reply:
Ok, I feel going from no max size to 1MB might be a big change for customers. I vote for 10MB as that might cover the majority of use-cases without a need for log rotation.

Reply:
Assuming we need 10,000 domains at 50K each at the default INFO level (since a simple inactive domain yields a "few k" of log data), gives us 50 MB.  10MB X 7 'feels' good enough to me.  On the other hand, for the logStash case, there's the chance of lost log data for a many-domain or a verbose logging setup if 'logStash' is started too late after the operator starts.

Reply:
Unless 1,000 domains is the target? I which case 10MB X 7 looks about right to me...

Reply:
How about 20MB x 10?

Reply:
Close enough for me.